### ISSUE TITLE
pytorch version 0.3 is not compatible with the code provided.

### ISSUE THREAD

**Issue by etema19** [Wanggcong/Spatial-Temporal-Re-identification#3]
Hey, thanks for your contribution. I can't reproduce your result due to version compatibility, as you explicitly described the code works on pytorch 0.3 but it has some features which are from pytorch 0.4 and this cause an error during compilation. For instance,  in train_market.py line 202: running_loss += loss.item(), this feature is not available in pytorch 0.3. Would you please advice me on how to tackle this issue or provide an alternative code which works? 

Thanks

**Comment by Wanggcong** [comment]
@etema19 Yes, this may be attributed to the version compatibility. I suggest you partially refer to [this work](https://github.com/layumi/Person_reID_baseline_pytorch/tree/master/tutorial) that considers several versions of pytorch.  Meanwhile, I think  pytorch 0.4 is ok.  I also doubt if pytorch 0.3 is a mistake in the readme.md because the pytorch version on my server is 0.4 now. And I do not attempt to degrade it into 0.3 for checking this bug.

**Comment by etema19** [comment]
Thanks for your reply, but, I have tried it on pytorch 0.4 as well, it didn't work since part of the code is with pytorch 0.3.
